apis/management.cattle.io/v3/rke_types.go

```diff
@@ -237,6 +237,8 @@ type AuthnConfig struct {
 	Strategy string `yaml:"strategy" json:"strategy,omitempty"`
 	// Authentication options
 	Options map[string]string `yaml:"options" json:"options,omitempty"`
+	// List of additional hostnames and IPs to include in the api server PKI cert
+	Sans []string `yaml:"sans" json:"sans,omitempty"`
```

Comment:
nit: This should probably be `SANs` not `sans` as SAN is an acronym.

Reply:
Which part? -- Can't be the yaml, as we have nothing else that has any capitalization in cluster.yml. Maybe the variable?

Reply:
Sorry, yes the variable.  Just to follow go naming conventions.  Go capitalizes acronyms.